.gitignore

```diff
@@ -5,7 +5,7 @@
 ## Secrets
 
 Nos/Assets/ProductionSecrets.xcconfig
-Nos/Assets/StagingSecrets.xcconfig
+Nos/Assets/DevSecrets.xcconfig
```

Comment:
If I'm reading this right, ProductionSecrets will be production, of course, then DevSecrets will be development. But then Secrets will be staging -- is that right? I think it's be nice to keep ProductionSecrets and DevSecrets and then have StagingSecrets if I'm understanding this correctly.

Reply:
I added StagingSecrets again. I guess I wanted to show that, in real life, secrets for production and staging are the same because at the moment we don't have anything we want to differentiate. But you're right, let's have different secrets so that we set the basis for future additions.